### PR TITLE
Handle validation flakes

### PR DESCRIPTION
**Pull request by kdorosh**

### CODE REVIEW

File: test/kube2e/gateway/gateway_suite_test.go [hunk 1]
```diff
@@ -8,9 +8,9 @@ import (
 	"testing"
 	"time"
 
-	v1 "github.com/solo-io/gloo/projects/gloo/pkg/api/v1"
-
 	"github.com/solo-io/gloo/pkg/cliutil/install"
+	gatewayv1 "github.com/solo-io/gloo/projects/gateway/pkg/api/v1"
+	v1 "github.com/solo-io/gloo/projects/gloo/pkg/api/v1"
```
Comment (marcogschmidt) [inline test/kube2e/gateway/gateway_suite_test.go]: Looks like this may fail on goimports check?

Comment (marcogschmidt) [reply test/kube2e/gateway/gateway_suite_test.go]: Evidently not 👍

**Comment by kdorosh** [comment]
/kick phantom nil https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=37c84885-07e9-4510-835b-888116dc38eb

**Comment by kdorosh** [comment]
/kick nil again https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=33602b8f-4ff5-4c8b-8744-6e4248a7df59

**Comment by kdorosh** [comment]
/kick worth looking into once i have time https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=f60b2bb6-0cf9-4b8b-a411-0c0d94654690

**Comment by kdorosh** [comment]
closing as stale, haven't seen any validation related flakes in Gloo

**Pull request closed without merging**